dataset/table.py

```diff
@@ -293,6 +293,24 @@ def _args_to_clause(self, args, clauses=()):
                 clauses.append(false())
             elif isinstance(value, (list, tuple)):
                 clauses.append(self.table.c[column].in_(value))
+            elif isinstance(value, dict):
+                key = list(value.keys())[0]
+                if key in ['like']:
```

Comment:
The contains check should be done against a tuple (there's no need for a list object). Of course, this also applies for all checks below. Don't forget trailing comma on single element tuples.

Reply:
just curious: whats the benefit in using tuples?

Reply:
They are faster and use less memory. Doesn't matter too much here, but since there's no downside, I would advise to use tuples. [Details in this stackoverflow answer](https://stackoverflow.com/questions/1708510/python-list-vs-tuple-when-to-use-each#1708538)

Reply:
kk, thx, i'll fix that tomorrow